addons/stock/wizard/stock_quant_relocate.py

```diff
@@ -0,0 +1,71 @@
+# -*- coding: utf-8 -*-is_not_full_package
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import fields, models, api
+from ast import literal_eval
```

Comment:
[Same reason](https://www.odoo.com/documentation/master/contributing/development/coding_guidelines.html#imports)
```suggestion

from ast import literal_eval

from odoo import fields, models, api
```